### PR TITLE
Update FAQs in faq.md

### PR DESCRIPTION
**Pull request by avoidthehack**
Clarified existing FAQs and added new FAQs. Linked out to key pages on the main LinkStack website.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -2,26 +2,47 @@
...
 
 Many social media platforms only allow you to add one link, with this you can simply link to your LinkStack page and have all the links you want displayed on one site. You can share all your links to your social media platform or 
-important links to easy accessible and hosted on your own web-server or web-hosting provider. On this website, other users can register and create their own links, you can access other user via the Admin Panel.
+important links to easy accessible and hosted on your own web-server or web-hosting provider. On this website, other users can register and create their own links, you can access other users via the Admin Panel.
 
 
 ## What do I need to use LinkStack?
 
-To host your own instance of LinkStack, you will need a web server running at least PHP 8.0.
+To host your own instance of LinkStack, you will need a web server running at least PHP 8.1
```
Comment (MagicLike) [inline docs/faq.md:23]: The full stop is missing here

Comment (MagicLike) [inline docs/faq.md]: The full stop is missing here

File: docs/faq.md [hunk 2]
```diff
@@ -2,26 +2,47 @@
...
 
 No problem! We also have a Docker version available.
+
 Do you want to use your own Domain or want a private instance? Sponsor LinkStack on GitHub and get free access to your own private instance.
+
+## Do I have to host LinkStack myself?
+
+Self-hosting is not the only option available for using LinkStack. 
+
+The project offers various tiers of [hosted instances](https://linkstack.org/hosted/) for GitHub Sponsors. Additionally, you can find [instances of LinkStack](https://linkstack.org/instances/) hosted by others in the community.
```
Comment (MagicLike) [inline docs/faq.md]: Please also remove the trailing slashes ^^

**Review by MagicLike — CHANGES_REQUESTED** [review]
LGTM, there are only a few small corrections to be made :+1:

File: docs/faq.md (hunk 1 above)
Comment (MagicLike) [reply docs/faq.md]: still missing ^^

Comment (avoidthehack) [reply docs/faq.md]: I added and changed it back to 8.0 (per a conversation with the main maintainer last month.)

I also linked out to the newly proposed access admin page.

**Pull request merged**